apps/vr-tests/screener.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   storybookConfigDir: '.storybook',
   apiKey: process.env.SCREENER_API_KEY,
   resolution: '1024x768',
-  baseBranch: (process.env.TRAVIS_PULL_REQUEST !== 'false' && process.env.TRAVIS_BRANCH) || 'master',
+  baseBranch: (process.env.TRAVIS_PULL_REQUEST !== 'false' && process.env.TRAVIS_BRANCH) || 'fabric-7',
```

Comment:
not sure this is what you want to be doing.

Reply:
Yeah... we are trying to create new baselines which will not override the current ones. In the 5.0 branch in place of `master` we have `5.0` so I believe this should be done for `fabric-7` branch  so we can have a separate set of baselines we can then compare against....